common/ideas/_economic.txt

```diff
@@ -14,46 +13,46 @@ ideas = {
 					limit = { has_country_flag = GRI_junta_established }
 					NOT = { has_completed_focus = GRI_the_war_economy }
 				}
-				if = {
+				else_if = {
 					limit = { has_completed_focus = FEA_hermans_stratocracy }
 					NOT = { has_completed_focus = FEA_retool_the_industrial_sector }
 				}
-				if = {
+				else_if = {
```

Comment:
These should not be else ifs

Reply:
Are you sure? Both focuses are in mutually exclusive trees, so how can they be completed at the same time?